modules/gateways/bitpaycheckout/bitpaycheckout_ipn.php

```diff
@@ -36,17 +37,19 @@ function checkInvoiceStatus($url){
     return $result;
 }
 
-$event = json_decode(file_get_contents("php://input"), true);
+$all_data = json_decode(file_get_contents("php://input"), true);
+$data = $all_data['data'];
```

Comment:
instead of $all_data I'd put $response since that's the actual response content

Reply:
Sure, Done